sphinx/util/jsdump.py

```diff
@@ -54,22 +54,14 @@ def decode_string(s: str) -> str:
     return ESCAPED.sub(lambda m: eval('"' + m.group() + '"'), s)
 
 
-reswords = set("""\
-abstract   else   instanceof   switch
-boolean   enum   int   synchronized
-break   export   interface   this
-byte   extends   long   throw
-case   false   native   throws
-catch   final   new   transient
-char   finally   null   true
-class   float   package   try
-const   for   private   typeof
-continue   function   protected   var
-debugger   goto   public   void
-default   if   return   volatile
-delete   implements   short   while
-do   import   static   with
-double   in   super""".split())
+reswords = {"abstract", "else", "instanceof", "switch", "boolean", "enum", "int",
```

Comment:
As per my earlier comment, please revert this one.

A

Reply:
is there some way to determine which modules are 'deprecated'?

Reply:
I think this is the only instance in this PR, in the general deprecations list also contains deprecated modules though.

A